src/TypeSpec.Extension/Emitter.Csharp/src/type/input-type.ts

```diff
@@ -62,8 +68,11 @@ export interface InputOffsetDateTimeType extends InputDateTimeTypeBase {
 
 export interface InputDurationType extends InputTypeBase {
     Kind: "duration";
+    Name: string;
     Encode: DurationKnownEncoding;
     WireType: InputPrimitiveType;
+    BaseType?: InputDurationType;
```

Comment:
we are not handling the newly added `BaseType` in generator?

Reply:
We do not take it right now because we do not have a case yet.
We have it here, because the input types should be identical to the type system provided by TCGC

Reply:
For more details, please refer the description of this PR: https://github.com/Azure/typespec-azure/pull/1015